wordclock/content/wificfg/clockcfg.html

```diff
@@ -19,74 +19,26 @@
 "<legend>Woordklok Configuratie</legend>"
 "<dl class=\"dlh\">"
-"<dt><label for=\"co\">Kleur</label></dt>"
-"<dd><select id=\"co\" name=\"cl_color\">"
-"<option class=\"white\" value=\"0\"",
-">white</option>"
-"<option class=\"bitPurple\" value=\"1\"",
-">bitPurple</option>"
-"<option class=\"purple\" value=\"2\"",
-">purple</option>"
-"<option class=\"magenta\" value=\"3\"",
-">magenta</option>"
-"<option class=\"pink\" value=\"4\"",
-">pink</option>"
-"<option class=\"red\" value=\"5\"",
-">red</option>"
-"<option class=\"orange\" value=\"6\"",
-">orange</option>"
-"<option class=\"yellow\" value=\"7\"",
-">yellow</option>"
-"<option class=\"yellowGreen\" value=\"8\"",
-">yellowGreen</option>"
-"<option class=\"green\" value=\"9\"",
-">green</option>"
-"<option class=\"blueGreen\" value=\"10\"",
-">blueGreen</option>"
-"<option class=\"cyan\" value=\"11\"",
-">cyan</option>"
-"<option class=\"lightBlue\" value=\"12\"",
-">lightBlue</option>"
-"<option class=\"blue\" value=\"13\"",
-">blue</option>"
-"<option class=\"rainBow\" value=\"14\"",
-">rainbow</option>"
-"</select></dd>"
+"<dt><label for=\"letterColorId\">Kleur</label></dt>"
+"<dd><input id=\"letterColorId\" type=\"text\" name=\"cl_color\" size=\"6px\""
+"value=\"",
+"\">"
+"<input id=\"letterButton\" type=\"button\" value=\"select\"></dd>"
+"<div id=\"letterPopUp\" class=\"popUpBox\">"
+"<div id=\"letter-picker\" class=\"popUpContent\"></div>"
+"</div>"
 "<dt><label for=\"ch\">Helderheid</label></dt>"
 "<dd><input id=\"ch\" type=\"number\" size=\"1\" min=\"0\" max=\"5\" step=\"1\" "
 "name=\"cl_brightness\" placeholder=\"1\" value=\"",
 "\"></dd>"
-"<dt><label for=\"co\">Achtergrondkleur</label></dt>"
-"<dd><select id=\"co\" name=\"cl_bgcolor\">"
-"<option class=\"white\" value=\"0\"",
-">white</option>"
-"<option class=\"bitPurple\" value=\"1\"",
-">bitPurple</option>"
-"<option class=\"purple\" value=\"2\"",
-">purple</option>"
-"<option class=\"magenta\" value=\"3\"",
-">magenta</option>"
-"<option class=\"pink\" value=\"4\"",
-">pink</option>"
-"<option class=\"red\" value=\"5\"",
-">red</option>"
-"<option class=\"orange\" value=\"6\"",
-">orange</option>"
-"<option class=\"yellow\" value=\"7\"",
-">yellow</option>"
-"<option class=\"yellowGreen\" value=\"8\"",
-">yellowGreen</option>"
-"<option class=\"green\" value=\"9\"",
-">green</option>"
-"<option class=\"blueGreen\" value=\"10\"",
-">blueGreen</option>"
-"<option class=\"cyan\" value=\"11\"",
-">cyan</option>"
-"<option class=\"lightBlue\" value=\"12\"",
-">lightBlue</option>"
-"<option class=\"blue\" value=\"13\"",
-">blue</option>"
-"</select></dd>"
+"<dt><label for=\"backColorId\">Achtergrondkleur</label></dt>"
+"<dd><input id=\"backColorId\" type=\"text\" name=\"cl_bgcolor\" size=\"6px\""
+    "value=\"",
+"\">"
+"<input id=\"backgroundButton\" type=\"button\" value=\"select\"></dd>"
+"<div id=\"backgroundPopUp\" class=\"popUpBox\">"
+"<div id=\"background-picker\" class=\"popUpContent\"></div>"
+"</div>"
```

Comment:
Als het makkelijk is mag je die color picker gewoon op de pagina zelf doen, dus geen popup

Reply:
Ik vind de popup eigenlijk wel wat netter, Ik moet een keer kijken of ik het kan reproduceren.